### ISSUE TITLE
Feature Request: ARM64 Support

### ISSUE THREAD

**Issue by Samuel12321** [ModernFlyouts-Community/ModernFlyouts#482]
Just Made this Issue to keep track of progress on this

With .net6 preview we are able to add native arm64 support

**Comment by Samuel12321** [comment]
Microsoft bug holding this [Here](https://github.com/dotnet/wpf/issues/4336)

**Comment by Samuel12321** [comment]
Can everyone upvote this so microsoft actually fixes it: https://developercommunity.visualstudio.com/t/ALINK-alexe-does-not-support-ARM64-Wi/1325460

**Comment by hmartinez82** [comment]
Just upvoted!

**Comment by hmartinez82** [comment]
@Samuel12321 Did you see the coments on https://developercommunity.visualstudio.com/t/ALINK-alexe-does-not-support-ARM64-Wi/1325460 on a workaround? I don't know if you are the same Samuel as Samuel G-S, the posts are private so I can't see what was the discussion

**Comment by Samuel12321** [comment]
yes that was me, i saw that and have implemented the changes. We ran into another problem with hostfxr.dll. This will be fixed as part of the next release. The team is quite busy at the moment so not sure when that will be, but hopefully soon.
